app/src/main/java/ch/blinkenlights/android/vanilla/FolderPickerActivity.java

```diff
@@ -66,7 +66,7 @@
 
 	@Override
 	public void onCreate(Bundle savedInstanceState) {
-		ThemeHelper.setTheme(this, R.style.BackActionBar);
+		ThemeHelper.setTheme(this, R.style.BackActionBar, this);
```

Comment:
I don't get why we need to pass both Context and Activity to setTheme: in any case we seem have access to the activity, so we could just change the signature of the function to expect an Activity instead of a Context.

Reply:
I need the Activity, so i passed it. I could extract the context from it and spare the context parameter

Reply:
Will check that and update it appropriately with the other styleissues you mentioned